docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+shipit:
+  build: ./
+  command: python kickoff-web.py -d sqlite:///update.db --username=admin --password=password --host=0.0.0.0 --port=5000 --verbose
```

Comment:
It's actually persistent, the `volumes` section below "mounts" current directory inside the image, so your db lives in `kickoff/update.db`